docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+---
+version: '3.4'
+
+services:
+
+  web:
+    build:
+      context: ./ccm_web/
+    environment:
+      - PORT=3000
+      - NODE_ENV=development
+    ports:
+      - 3001:3000
+      # Added to eliminate repeated console errors.
+      # I need to look into this more. Seems like only a dev environment problem.
+      # https://github.com/sveltejs/sapper-template/issues/175
+      - 10000:10000
+    volumes:
+      # Facilitates re-loading
+      - ./ccm_web/src:/app/src:z
+      - ./ccm_web/static:/app/static:z
```

Comment:
@pushyamig and @chrisrrowland, this should resolve the problem. Basically, the previous volume mapping seemed to be wiping out `node_modules` in `ccm_web`. The other files in `ccm_web` but not in `src` or `static` probably need a rebuild anyway when they're changed.